Makefile

```diff
@@ -25,10 +36,15 @@ E2E_BUILDAH_PVC_NAME ?= task-buildah
 # buildah task e2e test variables, image path, containerfile_path
 E2E_BUILDAH_CONTAINERFILE_PATH ?= /workspace/source/Dockerfile
 E2E_BUILDAH_IMAGE ?= test-buildah
-E2E_BUILDAH_REGISTRY ?= docker://registry.registry.svc.cluster.local:32222/test-buildah:latest
+E2E_BUILDAH_REGISTRY ?= registry.registry.svc.cluster.local:32222/test-buildah:latest
 E2E_BUILDAH_TLS_VERIFY ?= false
 E2E_BUILDAH_POPULATE_WORKSPACE ?= test/e2e/resources/populate-workspace-task.yaml
 
+#The local container registry to push the image during e2e testing of s2i-golang task
+E2E_S2I_IMAGE ?= registry.registry.svc.cluster.local:32222/test-s2i:latest
+E2E_S2I_TLS_VERIFY ?= false
```

Comment:
Please, add a top comment explaining why TLS verification needs to be false in combination with the internal container registry (during end-to-end).